<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>文泉驿正黑</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="cursor">
    <cursorShape>BlankCursor</cursorShape>
   </property>
   <widget class="QFrame" name="frame_Ptitile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1031</width>
      <height>91</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>876</x>
       <y>16</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="A66-appsrc.qrc">:/ico/ICO/LOGO.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_Picture">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>71</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="A66-appsrc.qrc">:/ICO/P1-PROGmini.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_Text">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>24</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>程序库</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>23</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="A66-appsrc.qrc">:/ico/ICO/P1-MSGmini.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_State">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>26</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>准备就绪</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>50</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="A66-appsrc.qrc">:/ico/ICO/P1-VERmini.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_Program">
     <property name="geometry">
      <rect>
       <x>914</x>
       <y>56</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_StateTip">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>27</y>
       <width>221</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_PumpTip">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>26</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>1031</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QToolButton" name="toolButton_B0">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿微米黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>程序库</string>
     </property>
     <property name="icon">
      <iconset resource="A66-appsrc.qrc">
       <normaloff>:/ico/ICO/P1-PROG.png</normaloff>:/ico/ICO/P1-PROG.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_B1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿微米黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>工步</string>
     </property>
     <property name="icon">
      <iconset resource="A66-appsrc.qrc">
       <normaloff>:/ico/ICO/P1-EDIT.png</normaloff>:/ico/ICO/P1-EDIT.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_B3">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿微米黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>手动</string>
     </property>
     <property name="icon">
      <iconset resource="A66-appsrc.qrc">
       <normaloff>:/ico/ICO/P1-MANU.png</normaloff>:/ico/ICO/P1-MANU.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_B4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿微米黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_B5">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿微米黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
     <property name="icon">
      <iconset resource="A66-appsrc.qrc">
       <normaloff>:/ico/ICO/P1-SET.png</normaloff>:/ico/ICO/P1-SET.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_B2">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>文泉驿微米黑</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>模具库</string>
     </property>
     <property name="icon">
      <iconset resource="A66-appsrc.qrc">
       <normaloff>:/ico/ICO/P1-MOLD.png</normaloff>:/ico/ICO/P1-MOLD.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>回零</string>
     </property>
     <property name="icon">
      <iconset resource="A66-appsrc.qrc">
       <normaloff>:/ICO/HomingMode.png</normaloff>:/ICO/HomingMode.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_Start">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>运行</string>
     </property>
     <property name="icon">
      <iconset resource="A66-appsrc.qrc">
       <normaloff>:/ico/ICO/P1-START.png</normaloff>:/ico/ICO/P1-START.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_4">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>10</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string>帮助</string>
     </property>
     <property name="icon">
      <iconset resource="A66-appsrc.qrc">
       <normaloff>:/ico/ICO/P1-HELP.png</normaloff>:/ico/ICO/P1-HELP.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>52</width>
       <height>52</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="A66-appsrc.qrc"/>
 </resources>
 <connections/>
</ui>
